src/components/ImageView/index.js

```diff
@@ -210,7 +213,7 @@ class ImageView extends PureComponent {
                         ...StyleUtils.getZoomSizingStyle(this.state.isZoomed, this.state.imgWidth, this.state.imgHeight, this.state.zoomScale,
                             this.state.containerHeight, this.state.containerWidth),
                         ...StyleUtils.getZoomCursorStyle(this.state.isZoomed, this.state.isDragging),
-                        ...this.state.isZoomed && this.state.zoomScale > 1 ? styles.pRelative : styles.pAbsolute,
+                        ...this.state.isZoomed && this.state.zoomScale >= 1 ? styles.pRelative : styles.pAbsolute,
```

Comment:
I'm curious why this change was needed

Reply:
when the zoom scale is 1 we don't need absolute position. also, it was one of the reasons for white screen 
because the image is loaded outside the screen.